cypress/integration/test.spec.tsx

```diff
@@ -0,0 +1,20 @@
+import * as Adapter from "enzyme-adapter-react-16";
+import * as React from "react";
+
+import { configure, shallow } from "enzyme";
+
+import { expect } from "chai";
+import { LeftNavComponent } from "../../src/components/left-nav";
+
+configure({ adapter: new Adapter() });
+
+context("Tests", () => {
+  it("TypeScript works", () => {
+    const x: number = 42;
+  });
+
+  it("Shallow renders work", () => {
+    const comp = shallow(<LeftNavComponent />);
+    expect(comp.text()).to.equal("Left Nav");
```

Comment:
Right now this file is mostly here to show functionality...I would imagine that ideally we'd want Enzyme tests to live near the components they're testing